tools/Vrops.py

```diff
@@ -128,6 +128,13 @@ def get_resources(self, target: str, token: str,
                     resource_object.uuid = resource["resource"]["identifier"]
                     resource_object.resourcekind = resource["resource"]["resourceKey"]["resourceKindKey"]
                     resource_object.parent = resource.get("relatedResources", [None])[0]
+                    resource_identifiers = resource.get('resource').get('resourceKey').get('resourceIdentifiers')
```

Comment:
This might fail because the second and the third `.get()` is executed on a `None`. You should provide the alternative.

Reply:
Absolutely right, corrected.